lib/Email.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable import/prefer-default-export */
+/**
+ * Email.js should be used for any common logic specific to emails.
+ */
+
+const EMAIL_REGEX = "([a-zA-Z0-9.!#$%&'+/=?^`{|}-][a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]*@[a-zA-Z0-9-]+?(\\.[a-zA-Z]+)+)";
+
+export {
+    EMAIL_REGEX,
+};
```

Comment:
Not sure why should we keep this here can we move CONST?  @cristipaval

Reply:
We already have this pattern for url links - https://github.com/Expensify/expensify-common/blob/main/lib/Url.js
So I followed the same pattern to create a file for email validations to be done.

Reply:
URL - https://github.com/Expensify/expensify-common/blob/main/lib/ExpensiMark.js#L76

Reply:
@Santhosh-Sellavel

Reply:
There are a few other options there and some computations are done. That's not the case here may be a const variable at the top should be good or should be moved to CONST.jsx.

cc: @cristipaval What do you think?

Reply:
Yes. I agree. I also created this file in the notion of more email features alone getting added to this file and that's why added this comment as well on "Email.js should be used for any common logic specific to emails.".
But if we strongly think new file is not required and move the same to CONST, I am ok to that as well.

Reply:
Hey there! Good catch @Santhosh-Sellavel !
@abdulrahuman5196 - Yes, I totally see why you added in a separate file trying to make it easy scalable. On the other hand, I think we should keep consistent with the existing approach (add it to `CONST.jsx`) and create a new file in the future if it will be the case to add more functionality.

Reply:
@cristipaval  @Santhosh-Sellavel 

Thank you for the feedback. I have moved the regex to CONST file. and update the PR as well. Kindly review and let me know if anything else is required from my end.